src/components/Admin/ManageSensors/DeleteSensorModal.tsx

```diff
@@ -61,8 +63,8 @@ const DeleteSensorModal: ({sensors}: DeleteSensorModalProps) => JSX.Element = ({
   // --------------- End state maintenance variables ------------------------
 
   const readyToSubmit =
-    purpleAirId !== '' &&
```

Comment:
should this be replaced with a check `purpleAirId !== Number.NaN`, or was this an extraneous check that can be safely removed?

Reply:
I basically just checked that the confirmPurpleAirId (which is a string from the user input) is not the empty string and then I compare the purpleAirId to the number converted confirmPurpleAirId